hangman.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'random-word'
+
+class Hangman
+  @word = ''
+  @guessedChars = []
+
+  def initialize
+    @word = RandomWord.nouns.next
+    @unmaskedWord = mask
+    @attempts = 6
+  end
+
+  def word
+    @word
+  end
+
+  def mask
+    @word.chars.map { |c| '_'}.to_s
```

Comment:
There's a method on String objects called gsub, which takes 2 arguments. The first is a regular expression, defining what to substitute. The second is the thing you want to substitute the regular expression with.
So:
```ruby
"foo".gsub(/./, "_") #=> "___"
```
Where `/./` means any single character.